api/taskmaster/domain/tasks/entities.py

```diff
@@ -0,0 +1,75 @@
+from dataclasses import dataclass
+from datetime import date, time
+from enum import Enum
+from typing import Optional
+
+from taskmaster.core.entities import Entity
+
+
+@dataclass
+class Task(Entity):
+    """Represent a pending task on our list"""
+
+    class Status(str, Enum):
```

Comment:
Going to change this internal class for a slightly more complex solution using a composition pattern. 

The original idea was to make a single import when using entities, without having to import the additional Enum classes. I want to maintain that, but there are times when I just want to import the Enum without having to import the whole entity. The change will allow to use both types of imports when required.